### DISCUSSION TITLE
Ways of supporting this project

### DISCUSSION THREAD

**Discussion by patkepa** [hoeken/PsychicHttp#203] [General]
Hey!

I recently started using this library at my project, and I seem really invested into Psychic ecosystem now, since that's exactly what I was looking for in HTTP library for ESP32 😃 

Is there any way, for me to meaningfully contribute towards this project? I see the project is under a heavy development on v2 branch, but I'm not so sure of what I could put towards it, especially at power levels gap between me and @hoeken and @mathieucarbou doing great job here. Is there any work in regards documentation, or testing that could see some help or any simple  issues viable for first time contribution?

Cheers!

**Comment by mathieucarbou** [comment]
Hello,
We have v2 in development right now but it is been a long time since we didn't have time to put into the project.
I've come up with a task list in our Discord at https://discord.gg/TAQrTR3f9C.
Feel free to come and help :-)

**Comment by hoeken** [comment]
I would say the best way is to help us by testing the v2 branch and if you
can contribute then we are happy to take pull requests.

**Comment by hoeken** [comment]
The last couple months I was really busy as I live on a sailboat and we
were preparing for big offshore passages (US to Bermuda to Caribbean). I'm
now in the Caribbean and we will be transitioning to a more relaxed style
where I will have more time for coding (yay Starlink!)

My goal is to get v2 finished and released within the next month.

**Comment by patkepa** [comment]
Thanks for quick answer! I'll be happy to go more into the details at Discord then 😄
